inst/rmarkdown/templates/pihph/resources/template.tex

```diff
@@ -0,0 +1,285 @@
+\date{}
+
+\captionnamefont{\footnotesize\bfseries}
+\captiontitlefont{\footnotesize}
+
+\renewcommand\abstractnamefont{\normalfont\footnotesize\bfseries}
+\renewcommand\abstracttextfont{\normalfont\footnotesize}
+\setlength\absleftindent{1cm}
+\setlength\absrightindent{1cm}
+\setlength{\abstitleskip}{-17pt}
+\setbeforesecskip{-18pt}
+\setaftersecskip{6pt}
+\setsecheadstyle{\normalfont\bfseries}
+\setbeforesubsecskip{-18pt}
+\setaftersubsecskip{6pt}
+\setsubsecheadstyle{\normalfont\bfseries}
+\setbeforesubsubsecskip{-18pt}
+\setaftersubsubsecskip{6pt}
+\setsubsubsecheadstyle{\normalfont\bfseries}
+\renewenvironment{quote}{\list{}{\leftmargin=0.7cm\rightmargin=0.7cm}\item[]\footnotesize}{\endlist}
+
+\setlrmarginsandblock{4cm}{4cm}{*}
+\setulmarginsandblock{4.5cm}{4.5cm}{*}
+\checkandfixthelayout
+
+\setlength{\footmarkwidth}{0em}
+\setlength{\footmarksep}{0em}
+\footmarkstyle{\textsuperscript{#1}\hspace{.3em}}
+\setfootins{24pt}{\bigskipamount}
+\renewcommand*{\footnoterule}{%
+   \kern-3pt%
+   \hrule width 0.4\columnwidth
+   \kern 2.6pt
+ \vspace{12pt}}
+
+\usepackage[usenames]{xcolor}
+\definecolor{pihphgreen}{RGB}{118, 146, 60}
+\usepackage[most]{tcolorbox}
+
+\newtcolorbox{tcdoublebox}[1][]{%
+  enhanced jigsaw,
+  sharp corners,
+  colback=white,
+  text width=.95\textwidth,
+  before={\vspace*{18pt}\noindent},
+  left skip=-5mm,
+  borderline={1pt}{-2pt}{black},
+  #1
+}
+
+\newcommand\commentsinvited{
+\begin{tcdoublebox}
+
+\section*{\textcolor{pihphgreen}{Comments invited}}
+
+\textit{PiHPh} relies on post-publication review of the papers that it publishes. If you have any comments on this piece, please add them to its comments site. You are encouraged to consult this site after reading the paper, as there may be comments from other readers there, and replies from the author. This paper's site is here:
+
+\vspace*{1ex}\noindent \url{\thisdoilink}
+
+\end{tcdoublebox}
+}
+
+\widowpenalty=10000
+\clubpenalty=10000
+
+% === START OF USER CUSTOMIZATION
+
+
+\title{$title$$if(subtitle)$:\\$subtitle$$endif$}
+\author{$for(author)$$author.name$\\\affiliation{$author.affiliation$}$sep$ \and $endfor$}
+
+\authorheader{$if(author-header)$$author-header$$else$$for(author)$$author.name$$sep$, $endfor$$endif$}
+\titleheader{$if(title-header)$$title-header$$else$$title$$endif$}
+
+$if(default-fonts)$
+\setmainfont{Cambria}
+\setlicensefont{Calibri}
+\setlogofont{Lekton}
+$endif$
+$if(alt-fonts)$
+\setmainfont{Caladea}
+\setlicensefont{Carlito}
+\setlogofont{Lekton}
+$endif$
+$if(main)$
+\setmainfont{$main$}
+$endif$
+$if(licensefont)$
+\setlicensefont{$licensefont$}
+$endif$
+$if(logofont)$
+\setlogofont{$logofont$}
+$endif$
+
+
+% If you do not have access to Cambria, you can use the metric-equivalent Caladea: https://fontlibrary.org/en/font/caladea
+% If you do not have access to Calibri, you can use the metric-equivalent Carlito: https://fontlibrary.org/en/font/carlito
+% Lekton available at https://www.fontsquirrel.com/fonts/lekton
```

Comment:
I would move this comment above before `alt-font` as this explains it

Reply:
b7e91b8